src/cm1.F

```diff
@@ -2259,11 +2259,12 @@ program cm1
       allocate( reqt(d3t) )
       reqt = 0
 
+      nparcelsActiveMax = 1e6   !The total allowable number of particles on each processor
 #ifdef MPI
-      padding = max(pdata_buffer*nparcels,100.0)
-      nparcelsLocal = ceiling((nparcels*1.0/numprocs)+padding)
+      padding = max(pdata_buffer*nparcelsActiveMax,100.0)
```

Comment:
By making this change, the value of `padding` is now independent of the `nparcels` value. I think a sanity check is needed to make sure that `nparcels/numprocs` is always smaller than `nparcelsActiveMax`.

Reply:
Good point. The padding should actually be a percentage of `nparcelsActiveMax`, since otherwise the padding would need to be adjusted throughout the simulation (which doesn't seem ideal/possible). So maybe I can change to `nparcelsActiveMax/numprocs` instead?

Reply:
Thanks @drichte2 . It is up to you to decide the calculation of `padding`. My point is that we need to check the value of `nparcels` here. For example, if I set `nparcels = 1 billion`, each MPI rank will have more than `nparcelsActiveMax` droplets and ideally the program should stop with an error message. This is missing from the current implementation.

Reply:
I just added a quick check to see if `nparcels` exceeds `nparcelsActiveMax`. @johnmauff , it's not setting the same thing twice. `nparcels` is the number of droplets to start the simulation with. `nparcelsActiveMax` is a maximum upper limit on the number of parcels that can ever be active at the same time. This would be something that should only be changed with architecture/memory limitations

Reply:
@drichte2  Historically nparcels was the total number of particles.  I think the current use is a bit confusing and error prone.   Maybe we need to have two namelist variables:  
 
nparcelsInit:  Initial number of particles
nparcelsMax:  Maximum number of particles.  

Might want to get @george-bryan input as we are proposing a namelist change.

Reply:
I agree with the last part -- I was trying to do this in a way that didn't mess with the namelist file. I'd be fine adding it at some point, but didn't want to mess up everyone's namelists